src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,33 @@
+.calendar {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 710px;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-block-end: $margin-day;
+    margin-inline-end: $margin-day;
+    width: $size-day;
+    height: $size-day;
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px solid black;
+    transition-duration: 0.5s;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+  }
+
+  &__day:active {
```

Comment:
You don't need to use `:active` state. This state is for `hover` too. 
Move these rules to the `__day:hover` selector, please.

![image](https://user-images.githubusercontent.com/25587299/221603868-7b98413c-6052-4dc9-a289-7e028d5c0eaa.png)